example/lib/pages/welcome_page.dart

```diff
@@ -77,6 +78,9 @@ class _WelcomePageState extends State<WelcomePage> {
       _buildExampleElementWidget("Controls configuration", () {
         _navigateToPage(ControlsConfigurationPage());
       }),
+      _buildExampleElementWidget("Custom Controls Player", () {
```

Comment:
Move it to the bottom of the list. In the future I will re-order list of examples based on priority.